fmc_rest_client/resources/access_policy.py

```diff
@@ -72,4 +72,9 @@ def __init__(self, name=None, container=None):
         self.destinationPorts = { 'objects': [] }
 
 
+class IntrusionPolicy(PolicyResource):
+
+    def __init__(self, name=None):
+        super().__init__(name)
+        self.name = name
```

Comment:
Not required to set name again as its already set in parent class constructor.  You will be still able to access self.name